pgjdbc/src/main/java/org/postgresql/core/Tuple.java

```diff
@@ -1,18 +1,29 @@
 /*
- * Copyright (c) 2008, PostgreSQL Global Development Group
+ * Copyright (c) 2017, PostgreSQL Global Development Group
```

Comment:
Please leave the original year as we only update the year for new files. Essentially says this has been copyright since 2008

Reply:
This is a new file, I think `2008` was a copy-paste mistake: https://github.com/pgjdbc/pgjdbc/pull/1701/files#diff-d9c999c9f4ed04929d4c670bba4d1d9e

Reply:
it is set to `2020` (in my commit, not in @tomdcc's original one, though) - wanted to keep his work intact, commit-wise

Reply:
On the final commit if done this year (I would hope) it should be 2020.